docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-working-with-analytics.md

```diff
@@ -6,7 +6,7 @@ template: back-office-user-guide-template
 related:
 ---
 
-Amazon QuickSight is a powerful business intelligence tool. It's highly customizable and has many tools that can't be covered in one document. This document describes a typical use case and how to start working with analytics, but doesn't cover all the editing options.
```

Comment:
I would not describe it with this negative sentiment but rather to focus on what it is.